From d3af16408346ddf0fa2bd0d2a086abe9e088e1af Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 5 Jul 2022 15:56:59 +0800
Subject: [PATCH] s32g: evb: usb: remove usb phy device node

We don't need usb phy device node anymore since the usb controller
driver in u-boot will handle usb phy initialization via ulpi interface.

Upstream-Status: Pending

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 fdts/s32gxxxa-evb.dtsi | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/fdts/s32gxxxa-evb.dtsi b/fdts/s32gxxxa-evb.dtsi
index 0fda509ef..2d23281ea 100644
--- a/fdts/s32gxxxa-evb.dtsi
+++ b/fdts/s32gxxxa-evb.dtsi
@@ -17,15 +17,9 @@
 			   <&clks S32G_SCMI_CLK_USB_LOW>;
 		pinctrl-0 = <&pinctrl0_usb &pinctrl1_usb>;
 		pinctrl-names = "default";
-		phys = <&saf1508_phy>;
+		phy_type = "ulpi";
 		dr_mode = "host";
 	};
-
-	saf1508_phy: usb_phy@44064000 {
-		compatible = "nxp,saf1508bet";
-		reg = <0x0 0x44064000 0x0 0x200>;
-		#phy-cells = <0>;
-	};
 };
 
 &usdhc0 {
-- 
2.36.1

